app/src/main/kotlin/no/nav/k9punsj/CoroutineRequestContext.kt

```diff
@@ -114,7 +114,7 @@ private fun Routes(
     }
     onError<Throwable> { error, serverRequest ->
         val exceptionId = serverRequest.headers().header(CALL_ID_KEY).firstOrNull() ?: UUID.randomUUID().toString()
-        logger.error("Ukjent feil med id $exceptionId . URI: ${serverRequest.uri()}", error)
+        logger.warn("Ukjent feil med id $exceptionId . URI: ${serverRequest.uri()}", error)
```

Comment:
Burde ikke ukjente feil fortsatt logges som error?
Det er vel spesifikt valideringsfeil man ikke vil logge error på. 
Kanskje lage en Exception håndtering for det?

Reply:
Vi har jo ikke gjort noe med ukjente feil de siste tre åren uansett så det gir ikke helt mening :)

Men ja, bedre exception håndtering er jo i backloggen for punsj :)

Reply:
Spennende att metrics-testen feiler her :D